### ISSUE TITLE
❓ Does Anomalib support running multiple training jobs in parallel?

### ISSUE THREAD

**Issue by joaopdrm** [open-edge-platform/anomalib#3198]
### Your Question

Hi, I’m using Anomalib v1.2.0 and I have a server with multiple GPUs.
I would like to train several models in parallel (one training process per GPU).

Before I implement my own job scheduler, I’d like to confirm:
	•	Does Anomalib officially support running multiple training processes in parallel?
	•	Are there any known issues regarding parallel runs (logging, checkpointing, dataloaders, multiprocessing, etc.)?
	•	Is there any recommended way to isolate resources per process (for example, GPU assignment or Lightning Trainer settings)?

### Forum Check

- [x] I have searched the discussions forum for an answer to my question.

**Comment by ashwinvaidya17** [comment]
1. Multi-gpu training was introduced in v2.2.0. However, if you are still keen on using v1.2.0 maybe installing from source might be an idea? You can then make similar changes to the codebase https://github.com/open-edge-platform/anomalib/pull/2435
2. There are no known issues in the main branch (but this won't meet the v1.2.0 req)
3. Since anomalib uses pytorch lightning under-the-hood, you can select which device you want to run your training job on. This is done by passing device id to the devices param, `devices=[1]` https://lightning.ai/docs/pytorch/stable/accelerators/gpu_basic.html#choosing-gpu-devices
Maybe the benchmarking script might interest you. https://github.com/open-edge-platform/anomalib/blob/7d6b89c173e98611dfbcfd0256647598c42dc002/src/anomalib/pipelines/benchmark/job.py#L64
https://anomalib.readthedocs.io/en/latest/markdown/guides/reference/pipelines/runners/parallel.html
